astropy/cosmology/core.py

```diff
@@ -93,92 +94,122 @@ class Cosmology(metaclass=ABCMeta):
         Arguments into the cosmology; used by subclasses, not this base class.
     name : str or None (optional, keyword-only)
         The name of the cosmology.
+    meta : dict or None (optional, keyword-only)
+        Metadata for the cosmology, e.g. a reference.
```

Comment:
OK, this is a major nitpick, but might as well get it right on first contribution...
```suggestion
        Metadata for the cosmology, e.g., a reference.
```
same for all other instances!